apps/prairielearn/assets/scripts/lib/questionsTable.js

```diff
@@ -185,6 +185,18 @@ onDocumentReady(() => {
         },
       },
     },
+
+    onResetView() {
+      const searchInputs = document.querySelectorAll(
+        '#questionsTable .form-control, #questionsTable .form-select',
+      );
+      searchInputs.forEach((searchInput) => {
+        searchInput.setAttribute(
+          'aria-label',
+          `Filter by ${searchInput.closest('th').dataset.field.replace(/_/g, ' ')}`,
```

Comment:
This produces labels like `Filter by assessment 1`. Rather than reading `data-field`, consider instead looking at the `textContent.trim()` property of the `div.th-inner`, which actually contains the human-readable column name.

Reply:
Updated to use `closest('th').firstElementChild.textContent.trim()`. I was unable to select the `div.th-inner` directly as it is not a parent of the select element. I instead opted to grab the `<th>` and use the first child which is `div.th-inner` for all columns.

Reply:
That seems unnecessarily brittle. Why not `closest('th').querySelector('div.th-inner').textContent.trim()`?

Reply:
That does make more sense. Updated this in [41ae7f8](https://github.com/PrairieLearn/PrairieLearn/pull/10514/commits/41ae7f8e62b0ad0ee356b109a7cc9320322f77ec).